src/wui/sound_options.cc

```diff
@@ -129,7 +129,14 @@ SoundOptions::SoundOptions(UI::Panel& parent, UI::SliderStyle style)
              style == UI::SliderStyle::kFsMenu ? UI::PanelStyle::kFsMenu : UI::PanelStyle::kWui,
              0,
              0,
-             UI::Box::Vertical) {
+             UI::Box::Vertical),
+     custom_songset(
+        this,
+        UI::PanelStyle::kFsMenu,
+        {0, 0},
+        _("Play your own music ingame"),
+        _("Put music files named custom_XX.ogg (where XX are digits) into datadir/music"),
```

Comment:
This sounds like clicking this checkbox causes Widelands to "         _("Put music files named ... into &lt;directory&gt;" ;)
Also players should be encouraged to use their home directory for this instead of modifying the data directory.
Suggestion:
```suggestion
        _("You can play custom in-game music by placing your own music files in ‘<Widelands Home Directory>/music/custom_XX.ogg’ (where ‘XX’ are sequential two-digit numbers starting with 00)"),
```

Reply:
< > crashes the text renderer. Is there some way to escape it? Otherwise, I think it is a bug in text renderer (there is already some special handling for \\)

Reply:
https://github.com/widelands/widelands/blob/049650ffd342ca9c3520e3b9c5312ffde95ec5fe/src/graphic/text_layout.h#L54
replaces `<` with `&lt;` and `>` with `&gt;` which the text renderer should accept

Reply:
thx, done